src/client/client.cpp

```diff
@@ -46,13 +47,32 @@ Client::Client(boost::asio::io_context& io_context, GameConfig config) :
     gameState(GamePhase::TITLE_SCREEN, config)
 {
     cam = new Camera();
+
+    audioManager = new AudioManager();
```

Comment:
I think it would be good to move this audio manager setup code into the client's `init` function. The main reason for this is because I think once you pull in the UI branch's code it will display the loading screen after the constructor has been called, so the program flow would go like this:

Program starts
Client Constructor is called
    Load audio files
Client init function is called
    Display loading screen
    load the rest of the assets

When we probably want it like this

Program starts
Client Constructor is called
Client init function is called
    display loading screen
    load all of the files, including the audio

Reply:
Made some changes for this comment and the above one, lmk if it's better.